src/platforms/ruby/guides/rails/index.mdx

```diff
@@ -7,7 +7,7 @@ description: "Learn about using Sentry with Rails."
 
 In Rails, all uncaught exceptions will be automatically reported.
```

Comment:
If you have these code samples, you can remove the index file and use the common getting started. To do this, remove the index file, enable the config.yml file, and move the Params and sessions content to a new page (I'd suggest Parameters and Sessions as the title). We document how to refactor this page here https://docs.sentry.io/contributing/approach/write-getting-started/